_maps/map_files/RandomRuins/SpaceRuins/syndiecakesfactory.dmm

```diff
@@ -628,7 +628,7 @@
 	},
 /area/ruin/space/syndicakefactory)
 "wW" = (
-/obj/machinery/suit_storage_unit/syndicate,
+/obj/machinery/suit_storage_unit/mining,
```

Comment:
is this intended to be a normal mining suit? the base version isn't space proof, only the asteroid skinned one is.

Reply:
inside of this storage is asteroid one
![image](https://github.com/user-attachments/assets/a1cfd32f-77ad-4791-bc02-1e5865128ffc)